src/wwwroot/js/genpage/main.js

```diff
@@ -723,15 +725,16 @@ function appendImage(container, imageSrc, batchId, textPreview, metadata = '', t
     if (typeof container == 'string') {
         container = getRequiredElementById(container);
     }
-    let div = createDiv(null, `image-block image-block-${type} image-batch-${batchId == "folder" ? "folder" : (batchId % 2)}`);
+    container.dataset.numImages === undefined ? container.dataset.numImages = 1 : container.dataset.numImages++
```

Comment:
missing `;` here too

Reply:
also wait wtf is this an ternary statement pair? ow. Use if/else

Reply:
_alsoalso_ this variable doesn't need to be in a div dataset, it's just a generic freefloating variable

Reply:
Valid point actually, nevermind that comment then